java-checks/src/test/files/checks/ConfusingOverloadCheck.java

```diff
@@ -72,3 +72,15 @@ public void doSomething(Computer.Pear p) {}
   static class Pear{}
   static class Apple{}
 }
+
+
+class UnknownParam {
+  private static int method(UNKNOWN arg) {
+
+  }
+}
+class UnknownParamChild extends UnknownParam {
+  protected int method(UNKNOWN arg) {
+
+  }
+}
```

Comment:
Can this be enforced in https://github.com/SonarSource/sonar-developer-toolset maybe ?